test/dvfinterface.test.js

```diff
@@ -74,6 +80,39 @@ contract('DVFInterface', function (accounts) {
     assert.equal(tokenBalanceStarkEx.toString(), _1e18.mul(new BN(5)).toString(), 'Balance not deposited')
   })
 
+  it('registerAndMakeDeposit: transfers token to StarkEx if approved with permit', async function () {
+    const currentBlock = await web3.eth.getBlock("latest")
+    const chainId = await web3.eth.getChainId()
+    const quantizedAmount = 5000000
+    const amount = new BN(quantum).mul(new BN(quantizedAmount))
+    const deadline = currentBlock.timestamp + 1000
```

Comment:
can also use `moveForwardTime` in `utils` to test what happens if deadline is breached